selfdrive/car/gm/carcontroller.py

```diff
@@ -110,7 +113,8 @@ def update(self, CC, CS):
     lka_active = CS.lkas_status == 1
     lka_critical = lka_active and abs(actuators.steer) > 0.9
     lka_icon_status = (lka_active, lka_critical)
-    if self.frame % self.params.CAMERA_KEEPALIVE_STEP == 0 or lka_icon_status != self.lka_icon_status_last:
+    # SW_GMLAN not yet on cam harness
+    if CS.CP.networkLocation != NetworkLocation.fwdCamera and self.frame % self.params.CAMERA_KEEPALIVE_STEP == 0 or lka_icon_status != self.lka_icon_status_last:
```

Comment:
Since we did not include this in the Panda list of allowed tx when using the cam harness, OP needs to no try as well